piv/key.go

```diff
@@ -198,19 +198,29 @@ type verifier struct {
 }
 
 func (v *verifier) Verify(attestationCert, slotCert *x509.Certificate) (*Attestation, error) {
-	root := v.Root
-	if root == nil {
-		ca, err := yubicoCA()
+	var roots []*x509.Certificate
+
+	if v.Root != nil {
+		roots = append(roots, v.Root)
```

Comment:
nit: if this option is provided, don't modify it.

Reply:
addressed, thanks. now if the option is provided, it is used as is